metricflow-semantics/metricflow_semantics/model/semantics/linkable_element.py

```diff
@@ -324,15 +344,22 @@ class MetricSubqueryJoinPathElement:
         metric_reference: The metric that's aggregated in the subquery.
         join_on_entity: The entity that the metric is grouped by in the subquery. This will be updated in V2 to allow a list
             of entitites & dimensions.
+        entity_links: Sequence of entities joined to get from a metric source to the `join_on_entity`.
         metric_to_entity_join_path: Describes the join path used in the subquery to join the metric to the `join_on_entity`.
             Can be none if all required elements are defined in the same semantic model.
     """
 
     metric_reference: MetricReference
+    metric_derived_from_semantic_models: Tuple[SemanticModelReference, ...]
```

Comment:
Huh. This name is awkward. Can we just call this `derived_from_semantic_models`, too? Since this is a MetricSubqueryJoinPathElement it seems like it'll be ok. That also makes it a little easier for us to make this part of a common interface with the SemanticModelJoinPath if we decide to do so in the future.